vimeo-networking/src/main/java/com/vimeo/networking/utils/ISO8601.java

```diff
@@ -98,9 +102,6 @@ public static Date toDate(final String iso8601string) throws ParseException {
         // Account for ISO 8601 standard of using the character 'Z' to represent UTC
         String s = iso8601string.replace("Z", "+00:00");
         try {
-            if (s.charAt(22) != ':') {
-                throw new ParseException("Invalid ISO 8601 format", 0);
-            }
```

Comment:
This won't crash it, right? If we remove the wrong portion of the string below then dateFormat.parse() will probably throw a parse exception which this method already throws.